### ISSUE TITLE
Package not on CRAN anymore?

### ISSUE THREAD

**Issue by willgearty** [benmarwick/gsloid#2]
I use this package for some of the tests and examples for my package [deeptime](https://github.com/willgearty/deeptime). It seems that this package is no longer on CRAN, and CRAN is now threatening to take my package off of CRAN because of the dependency. Do you anticipate that gsloid will be restored on CRAN soon?

**Comment by benmarwick** [comment]
Hi Will, 

Thanks for your note, great to hear that the package is useful to you. Yes, I've been attempting to update it on CRAN over the last week or so, after I got an email from them that checks were returning a note: 'Package vignette with placeholder title 'Vignette Title'' I couldn't reproduce this note at all, but in updating and resubmitting it triggered some new notes that I've been working to remove. 

I've been logging my progress here: https://github.com/benmarwick/gsloid/blob/master/cran-comments.md. I've just submitted another update to CRAN a few hours ago. It has to be manually checked each time because their URL/DOI checker doesn't allow enough time for DOIs to resolve, and so I get a note during checks. 

Hopefully I am close to getting it back on CRAN! I will update you here as soon as I know anything.
